<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>612</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777193</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>仿宋</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>SPCCP-Miner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/Downloads/car.png</normaloff>:/Downloads/car.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16771</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QPushButton" name="start_pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/Downloads/运行程序.png</normaloff>:/Downloads/运行程序.png</iconset>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>        ∆t</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="timeSpanlineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  t-threshold</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="t_thresholdlineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>220</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  PI-threshold</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pi_thresholdlineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.5</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>771</width>
      <height>211</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'仿宋'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/Downloads/重置.png</normaloff>:/Downloads/重置.png</iconset>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>19</y>
      <width>511</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab{width:160}

QTabBar::tab{height:40}</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/Downloads/打开.png</normaloff>:/Downloads/打开.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Read Data Mode</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>511</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:10pt; color:#5555ff;&quot;&gt;//Please press File-&amp;gt;open button in menu to select .txt data file.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/Downloads/编辑 笔 记事 手写.png</normaloff>:/Downloads/编辑 笔 记事 手写.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Input Data Mode</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>511</width>
        <height>231</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:10pt; color:#5555ff;&quot;&gt;//Please input data below in the correct format.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Youtube help video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/Downloads/表格-保存.png</normaloff>:/Downloads/表格-保存.png</iconset>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Github source code</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionopen_2"/>
    <addaction name="actionsave"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actioninstruction"/>
    <addaction name="actionhelp_video"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionFile_Mode"/>
    <addaction name="actionWrite_Mode"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMode"/>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/Downloads/打开.png</normaloff>
     <normalon>../../../Downloads/打开.png</normalon>:/Downloads/打开.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionopen_2">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/Downloads/打开 (1).png</normaloff>:/Downloads/打开 (1).png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionopen_3">
   <property name="icon">
    <iconset>
     <normaloff>../../../Downloads/编辑 笔 记事 手写.png</normaloff>
     <normalon>../../../Downloads/编辑 笔 记事 手写.png</normalon>../../../Downloads/编辑 笔 记事 手写.png</iconset>
   </property>
   <property name="text">
    <string>input data</string>
   </property>
  </action>
  <action name="actionFile_Mode">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/Downloads/打开.png</normaloff>:/Downloads/打开.png</iconset>
   </property>
   <property name="text">
    <string>Read Data Mode</string>
   </property>
  </action>
  <action name="actionWrite_Mode">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/Downloads/编辑 笔 记事 手写.png</normaloff>:/Downloads/编辑 笔 记事 手写.png</iconset>
   </property>
   <property name="text">
    <string>Input Data Mode</string>
   </property>
  </action>
  <action name="actioninstruction">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/Downloads/说明书.png</normaloff>:/Downloads/说明书.png</iconset>
   </property>
   <property name="text">
    <string>instruction</string>
   </property>
  </action>
  <action name="actionhelp_video">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/Downloads/视频 (1).png</normaloff>:/Downloads/视频 (1).png</iconset>
   </property>
   <property name="text">
    <string>help video</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/Downloads/保存.png</normaloff>:/Downloads/保存.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
